tests/test_integration.py

```diff
@@ -101,7 +123,8 @@ def flatten_asset_children(asset_children):
     return flat_dict
 
 
-def check_upload_completion(client, download_folder_id, upload_folder_id):
+def check_upload_completion(setup_client, download_folder_id, upload_folder_id):
+    client = setup_client
     # Do a comparison against filenames and filesizes here to make sure they match
 
     print("Beginning upload comparison check")
```

Comment:
Now that we have the `xxHash64` field available on assets, we should testing for this rather than comparing against filenames and filesizes, as those are both passed client-side and don't actually represent the state of the submitted file.

This will require utilizing the same `xxhash64` Python library that Tom used for the back-end.